profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info

```diff
@@ -0,0 +1,260 @@
+features[user_permission][] = create newsroom_item content
+features[user_permission][] = delete any newsroom_item content
+features[user_permission][] = delete own newsroom_item content
+features[user_permission][] = edit any newsroom_item content
+features[user_permission][] = edit own newsroom_item content
+features[variable][] = additional_settings__active_tab_newsroom_item
+features[variable][] = comment_anonymous_newsroom_selection
+features[variable][] = comment_default_mode_newsroom_selection
+features[variable][] = comment_default_per_page_newsroom_selection
+features[variable][] = comment_form_location_newsroom_selection
+features[variable][] = comment_newsroom_selection
+features[variable][] = comment_preview_newsroom_selection
+features[variable][] = comment_subject_field_newsroom_selection
+features[variable][] = date_format_date_only
+features[variable][] = date_format_minimal
+features[variable][] = date_format_short_newsroom
+features[variable][] = field_bundle_settings_node__newsroom_item
+features[variable][] = field_bundle_settings_node__newsroom_selection
+features[variable][] = i18n_sync_node_type_newsroom_selection
+features[variable][] = language_content_type_newsroom_item
+features[variable][] = language_content_type_newsroom_selection
+features[variable][] = menu_options_newsroom_item
+features[variable][] = menu_options_newsroom_selection
+features[variable][] = menu_parent_newsroom_item
+features[variable][] = menu_parent_newsroom_selection
+features[variable][] = node_options_newsroom_item
+features[variable][] = node_options_newsroom_selection
+features[variable][] = node_preview_newsroom_item
+features[variable][] = node_preview_newsroom_selection
+features[variable][] = node_preview_page
+features[variable][] = node_submitted_newsroom_item
+features[variable][] = node_submitted_newsroom_selection
+features[variable][] = pathauto_node_newsroom_item_pattern
+features[views_view][] = newsletter_subscription
+features[views_view][] = newsroom
+features[views_view][] = newsroom_calendar_navigation
+features[views_view][] = newsroom_navigation
+features[views_view][] = newsroom_page_content
+features[views_view][] = newsroom_services
+features[views_view][] = newsroom_term_selection
+features[views_view][] = newsroom_type_featured_item
+features_exclude[dependencies][field_group] = field_group
+features_exclude[dependencies][cce_basic_config] = cce_basic_config
+features_exclude[dependencies][link] = link
+features_exclude[dependencies][media] = media
+features_exclude[dependencies][image] = image
+features_exclude[dependencies][list] = list
+features_exclude[dependencies][file] = file
+features_exclude[dependencies][email] = email
+features_exclude[dependencies][fullcalendar] = fullcalendar
+features_exclude[dependencies][entityreference] = entityreference
+features_exclude[dependencies][date] = date
+features_exclude[dependencies][nexteuropa_multilingual] = nexteuropa_multilingual
```

Comment:
actually the dependency is yes, can be removed, but the sites are using this feature by default because of the decision by dg comm regarding the multilingual behavior.